ci-operator/config/maistra/api/maistra-api-maistra-2.1.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.1"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
```

Comment:
Are we skipping integration tests?

Reply:
Yes, istio integration test is hanging when running on OpenShift. I created a JIRA about fixing that in a future sprint.